js/main.js

```diff
@@ -0,0 +1,14 @@
+function randomInteger(min,max){
+  if(min>=0 && max>=0 && min<max){
+    return Math.floor(Math.random() * (max - min + 1) ) + min;
+  }
+}
+randomInteger();
+
+function lengthCheck (line,maxLength) {
```

Comment:
Не забываем про пробелы
```suggestion
function lengthCheck (line, maxLength) {
```